packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -104,116 +111,139 @@ class DependencyExtractionWebpackPlugin {
 						}
 					}
 				}
+			}
 
-				const runtimeChunk = entrypoint.getRuntimeChunk();
+			const runtimeChunk = entrypoint.getRuntimeChunk();
 
-				const assetData = {
-					// Get a sorted array so we can produce a stable, stringified representation.
+			const assetData = {
+				// Get a sorted array so we can produce a stable, stringified representation.
 					dependencies: Array.from(
 						entrypointExternalizedWpDeps
 					).sort(),
-					version: runtimeChunk.hash,
-				};
-
-				const assetString = this.stringify( assetData );
-
-				// Determine a filename for the asset file.
-				const [ filename, query ] = entrypointName.split( '?', 2 );
-				const buildFilename = compilation.getPath(
-					compiler.options.output.filename,
-					{
-						chunk: runtimeChunk,
-						filename,
-						query,
-						basename: basename( filename ),
-						contentHash: createHash( 'md4' )
-							.update( assetString )
-							.digest( 'hex' ),
-					}
-				);
-
-				if ( combineAssets ) {
-					combinedAssetsData[ buildFilename ] = assetData;
-					continue;
+				version: runtimeChunk.hash,
+			};
+
+			const assetString = this.stringify( assetData );
+
+			// Determine a filename for the asset file.
+			const [ filename, query ] = entrypointName.split( '?', 2 );
+			const buildFilename = compilation.getPath(
+				compiler.options.output.filename,
+				{
+					chunk: runtimeChunk,
+					filename,
+					query,
+					basename: basename( filename ),
+					contentHash: createHash( 'md4' )
+						.update( assetString )
+						.digest( 'hex' ),
 				}
+			);
 
-				const assetFilename = buildFilename.replace(
-					/\.js$/i,
-					'.asset.' + ( outputFormat === 'php' ? 'php' : 'json' )
-				);
-
-				// Add source and file into compilation for webpack to output.
-				compilation.assets[ assetFilename ] = new RawSource(
-					assetString
-				);
-				runtimeChunk.files.push( assetFilename );
+			if ( combineAssets ) {
+				combinedAssetsData[ buildFilename ] = assetData;
+				continue;
 			}
 
-			if ( combineAssets ) {
-				// Assert the `string` type for output path.
-				// The type indicates the option may be `undefined`.
-				// However, at this point in compilation, webpack has filled the options in if
-				// they were not provided.
+			const assetFilename = buildFilename.replace(
+				/\.js$/i,
+				'.asset.' + ( outputFormat === 'php' ? 'php' : 'json' )
+			);
+
+			// Add source and file into compilation for webpack to output.
+			compilation.assets[ assetFilename ] = new RawSource( assetString );
+			if ( isWebpack4 ) runtimeChunk.files.push( assetFilename );
+			else runtimeChunk.files.add( assetFilename );
+		}
+
+		if ( combineAssets ) {
+			// Assert the `string` type for output path.
+			// The type indicates the option may be `undefined`.
+			// However, at this point in compilation, webpack has filled the options in if
+			// they were not provided.
 				const outputFolder = /** @type {{path:string}} */ ( compiler
 					.options.output ).path;
```

Comment:
This statement is incorrectly indented. Running Prettier on the file will solve it. The Static Analysis CI check is failing because of this.

Reply:
🤦 I had the formatter disabled in the editor. Addressed in d8e4832